.travis.yml

```diff
@@ -1,9 +1,15 @@
 ---
 git:
   depth: 1
-sudo: true # can't use this due to git upgrade requirement
 language: generic
 
+sudo: false
```

Comment:
Heads up, I think newly created repositories are always in container mode by default now, we could remove this I guess